sdk/src/main/java/com/mapbox/maps/viewannotation/ViewAnnotation.kt

```diff
@@ -0,0 +1,36 @@
+package com.mapbox.maps.viewannotation
+
+import android.view.View
+import android.view.ViewTreeObserver
+import android.widget.FrameLayout
+import java.util.*
+
+internal data class ViewAnnotation(
+  /**
+   * Inflated Android view.
+   */
+  val view: View,
+  /**
+   * We handle visibility automatically in the manager
+   * if user did not specify [ViewAnnotationOptions.visible]  explicitly  during add / update operation
+   */
+  var handleVisibility: Boolean,
```

Comment:
Maybe `handleVisibilityManually` / `handleVisibilityAutomatically`, not sure what does it mean now

Reply:
it's internal but I'll rename to make it clearer